docs/user-guide/logging/overview.md

```diff
@@ -0,0 +1,107 @@
+---
+assignees:
+- crassirostris
+- piosz
+
+---
+
+Application and systems logs can help you understand what is happening inside your cluster. The logs are particularly useful for debugging problems and monitoring cluster activity. Most modern applications have some kind of logging mechanism; as such, most container engines are likewise designed to support some kind of logging. The easiest and most embraced logging method for containerized applications is to write to the standard output and standard error streams.
+
+However, the native functionality provided by a container engine or runtime is usually not enough for a complete logging solution. For example, if a container crashes, a pod is evicted, or a node dies, you'll usually still want to access your application's logs. As such, logs should have a separate storage and lifecycle independent of nodes, pods, or containers; this concept is called __cluster-level-logging__. Cluster-level logging requires a separate back-end to store, analyze, and query logs. Kubernetes provides no native storage solution for logs data, but you can integrate many existing logging solutions into your Kubernetes cluster.
+
+In this document, you can find:
+
+* A basic demonstration of logging in Kubernetes using the standard output stream
+* A detailed description of the node logging architecture in Kubernetes
+* Guidance for implementing cluster-level logging in Kubernetes
+
+In the last section the assumption is that logging back-end is present inside or outside of the cluster. If you're not interested in having cluster-level logging, you might still find the description how logs are stored and handled on the node to be useful.
+
+## Basic logging in Kubernetes
+
+In this section, you can see an example of basic logging in Kubernetes that outputs data to the standard output stream. This demonstration uses a [pod specification](/docs/user-guide/logging/counter-pod.yaml) with a container that writes some text to standard output once per second.
+
+{% include code.html language="yaml" file="counter-pod.yaml" %}
+
+To run this pod, use the following command:
+
+```shell
+$ kubectl create -f ./counter-pod.yaml
+pod "counter" created
+```
+
+To fetch the logs, use the `kubectl logs` command, as follows
+
+```shell
+$ kubectl logs counter
+0: Tue Jun  2 21:37:31 UTC 2015
+1: Tue Jun  2 21:37:32 UTC 2015
+2: Tue Jun  2 21:37:33 UTC 2015
+3: Tue Jun  2 21:37:34 UTC 2015
+4: Tue Jun  2 21:37:35 UTC 2015
+5: Tue Jun  2 21:37:36 UTC 2015
+...
+```
+
+You can use `kubectl logs` to retrieve logs from a previous instantiation of a container, in case the container has crashed. If your pod has multiple containers, you should specify which container's logs you want to access. See the [`kubectl logs` documentation](/docs/user-guide/kubectl/kubectl_logs) for more details.
```

Comment:
How can I see logs from a previous run?
How should I specify which container's logs I want to see?